k8s/agent+helper/templates/_agent.tpl

```diff
@@ -0,0 +1,178 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
duplicated signature. Suggestion for the removal of this file.

Reply:
All functions in _agent.tpl separated and deleted the duplicate ones